ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource

```diff
@@ -69,10 +71,10 @@ Select Pipeline For The Run
     [Documentation]    Selects the given ${pipeline_name} from the pipeline
     ...                dropdown in the Run creation form.
     [Arguments]    ${pipeline_name}
-    Wait Until Page Contains Element    xpath://section[@id="run-section-pipeline"]//span[@class="pf-v5-c-dropdown__toggle-text"]/..
-    Click Element    xpath://section[@id="run-section-pipeline"]//span[@class="pf-v5-c-dropdown__toggle-text"]/..
-    Wait Until Page Contains Element    xpath://ul/li/a[text()="${pipeline_name}"]
-    Click Element     xpath://ul/li/a[text()="${pipeline_name}"]
+    Wait Until Page Contains Element    xpath://*[@id="pipeline-selector"]
+    Click Element    xpath://*[@id="pipeline-selector"]
+    Wait Until Page Contains Element    xpath://*[@id="run-section-pipeline"]/div[2]/div/div/div[1]/div/div/ul/div/table/tbody/tr/td[text()="${pipeline_name}"]
```

Comment:
Having hard-coded array index and full path here may be a fragile. Didn't look into the page source, thus, please, don't take me highly serious, but doesn't make sense to have this xpath relaxed a bit?

This comment applies to most of the rest updated xpaths too :slightly_smiling_face:

Reply:
@jstourac thanks for the feedback. will work on it